background/token-manager.js

```diff
@@ -203,4 +214,28 @@ const tokenMan = (() => {
     err.code = r.status;
     throw err;
   }
+
+  async function detectVivaldiWebRequestBug() {
```

Comment:
How about moving bug detection logic/state into `toolbox.js`? Therefore we can do something like:
```js
/* global hasVivaldiBug */ // toolbox.js

await webextLaunchWebAuthFlow({
  ...
  alwaysUseTab: await hasVivaldiBug()
})
```

Reply:
This detector is needed only for this webRequest bug. If someone reports a Vivaldi-specific bug in CSS/DOM/JS then of course we will move the detector. The probability of such a bug is very low, AFAICT, because Vivaldi is more about outer customization.